README.md

```diff
@@ -2,7 +2,7 @@
 
 # Godot Mod Loader Development Tool
 
-<img alt="Godot Mod Loader Devtool Logo" src="https://github.com/KANAjetzt/godot-mod-tools/assets/41547570/243089de-e5ab-4dbd-b215-d053f12ab0d5" width="256" />
+<img alt="Godot Mod Loader Devtool Logo" src="https://github.com/GodotModding/godot-mod-tool/blob/main/addons/mod_tool/icon.png" width="256" />
```

Comment:
i think the url needs to lead to raw.githubusercontent
like this https://raw.githubusercontent.com/Qubus0/diceRoller/main/godot/icon.png
otherwise it will get a page, not the raw image https://github.com/Qubus0/diceRoller/blob/main/godot/icon.png

Reply:
Tested it by setting the link to `https://github.com/KANAjetzt/godot-mod-tools/blob/chore_add_logo_to_repo/icon.png` - seems to work.

Reply:
okay cool